### PR TITLE
Feature/updates

### PR DESCRIPTION
**Pull request by krasich74**
A little bit of actualization of modules

Now we can use abstract presentable modules, and all modules use coordinators

### CODE REVIEW

File: touchin-configurable/Code/Assembly/module.swift.liquid [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import TICoordinatorKit
+import LeadKit
```
Comment (petropavel13) [inline touchin-configurable/Code/Assembly/module.swift.liquid:2]: import TISwiftUtils for closures?

File: touchin-custom-view/Code/View/view.swift.liquid [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import UIKit
+import LeadKit
+
+final class {{ module_info.name }}View: BaseView {
```
Comment (petropavel13) [inline touchin-custom-view/Code/View/view.swift.liquid]: BaseInitializeableView

File: touchin-custom-view/Code/View/view_controller.swift.liquid [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    // MARK: - Actions
+
+}
+
+// MARK: - Public
+
+extension {{ module_info.name }}ViewController {
+}
```
Comment (petropavel13) [inline touchin-custom-view/Code/View/view_controller.swift.liquid]: ViewController is hidden behind presentable

File: touchin-custom-view/Code/View/view_controller.swift.liquid [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+final class {{ module_info.name }}ViewController: BaseCustomViewController<{{ module_info.name }}ViewModel, {{ module_info.name }}View>, {{ module_info.name }}Module, DisposeBagHolder {
+
+    // MARK: - Overrides
+
+    // MARK: - Views
+
+    // MARK: - Properties
+
+    let disposeBag = DisposeBag()
```
Comment (petropavel13) [inline touchin-custom-view/Code/View/view_controller.swift.liquid:14]: not sure about rx in view controller
MVP-like arch looks better for me now

File: touchin-custom-view/Code/ViewModel/view_model.swift.liquid [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+// MARK: - Actions
+
+extension {{ module_info.name }}ViewModel {
+}
+
+// MARK: - Public
+
+extension {{ module_info.name }}ViewModel {
+}
```
Comment (petropavel13) [inline touchin-custom-view/Code/ViewModel/view_model.swift.liquid]: what the difference between actions and public?

File: touchin-custom-view/Code/View/view.swift.liquid (hunk 2 above)
Comment (krasich74) [reply touchin-custom-view/Code/View/view.swift.liquid]: I can't find it in scope) 

BaseView is a subclass of UIVIew, implementing InitializableView protocol

File: touchin-custom-view/Code/View/view_controller.swift.liquid (hunk 4 above)
Comment (krasich74) [reply touchin-custom-view/Code/View/view_controller.swift.liquid:14]: To my mind all of our current projects use rx in viewController, so I think it's just more convenient)

File: touchin-custom-view/Code/View/view_controller.swift.liquid (hunk 3 above)
Comment (krasich74) [reply touchin-custom-view/Code/View/view_controller.swift.liquid]: I don't get it..?

File: touchin-custom-view/Code/View/view.swift.liquid (hunk 2 above)
Comment (petropavel13) [reply touchin-custom-view/Code/View/view.swift.liquid]: as well as BaseView
none of them in LeadKit right now

File: touchin-custom-view/Code/View/view_controller.swift.liquid (hunk 3 above)
Comment (petropavel13) [reply touchin-custom-view/Code/View/view_controller.swift.liquid]: you can't call public method of VC coz your VC is behind Presentable protocol

Comment (krasich74) [reply touchin-custom-view/Code/View/view_controller.swift.liquid]: this is actually internal... I'd better rename it

**Review by petropavel13 — APPROVED** [review]

**Pull request merged**